R/splatPop-simulate.R

```diff
@@ -56,9 +56,9 @@ splatPopSimulate <- function(params = newSplatPopParams(nGenes = 1000),
                              sparsify = TRUE,
                              verbose = TRUE, ...) {
 
-    if (verbose) {message("Getting parameters...")}
+    if (verbose) {message("Designing population...")}
     params <- setParams(params, ...)
-    params <- expandParams(params)
+    #params <- expandParams(params)
```

Comment:
I guess the expansion got moved somewhere else? Delete the commented line if not needed here.